lib/webpacker/railtie.rb

```diff
@@ -89,4 +89,12 @@ class Webpacker::Engine < ::Rails::Engine
       end
     end
   end
+
+  initializer "webpacker.set_source" do
+    if defined?(Rails::Server) || defined?(Rails::Console)
```

Comment:
Why we need to care about the server or the console?

Reply:
I thought it would only be necessary for those scenarios.  I guess there would be no need to limit.